src/components/pages/TeamPage.jsx

```diff
@@ -20,66 +21,23 @@ const TeamPage = () => {
 
         <h2 className="font-spartan font-semibold text-2xl lg:text-3xl"> Team Mentors</h2>
         <div className="w-full flex flex-wrap items-center justify-center mb-4">
-          <MentorCard BlobImage={GreenBlobImage} teamMemberData={teamMembers[0]} imgShift="12" textShift="8"/>
-          <MentorCard BlobImage={YellowBlobImage} teamMemberData={teamMembers[1]} cardShift="2"/>
-          <MentorCard BlobImage={PurpleBlobImage} teamMemberData={teamMembers[2]} cardShift="6"/>
+          <MentorCard BlobImage={GreenBlobImage} teamMemberData={teamMembers[0]} cardShift="0" imgShift="24" textShift="8" />
+          <MentorCard BlobImage={YellowBlobImage} teamMemberData={teamMembers[1]} cardShift="2" />
+          <MentorCard BlobImage={PurpleBlobImage} teamMemberData={teamMembers[2]} cardShift="6" />
         </div>
 
         <h2 className="font-spartan font-semibold text-2xl lg:text-3xl mb-6">Team Members</h2>
         <div className="w-full flex flex-row flex-wrap items-center justify-center gap-10 mb-10">
-
-          <div className="w-60 basis-60 flex flex-col items-center gap-3">
-            <img
-              src={teamMembers[3].src}
-              alt={teamMembers[3].alt}
-              className="w-36 h-36 rounded-full border-solid border-4 border-darkGreen"
-            />
-            <p className="w-full font-spartan font-semibold text-xl lg:text-2xl text-center">{teamMembers[3].name}</p>
-            <p className="w-full font-spartan font-semibold text-lg lg:text-xl text-center">{teamMembers[3].title}</p>
-          </div>
-
-          <div className="w-60 basis-60 flex flex-col items-center gap-3">
-            <img
-              src={teamMembers[4].src}
-              alt={teamMembers[4].alt}
-              className="w-36 h-36 rounded-full border-solid border-4 border-darkGreen"
-            />
-            <p className="w-full font-spartan font-semibold text-xl lg:text-2xl text-center">{teamMembers[4].name}</p>
-            <p className="w-full font-spartan font-semibold text-lg lg:text-xl text-center">{teamMembers[4].title}</p>
-          </div>
+          {[...Array(teamMembers.length).keys()]
```

Comment:
This is a very clever way of chaining `map` onto `filter` so that you can return a JSX component. But it over complicates things. Because you are trying to return a JSX component it makes the use of `filter` a little more involved since you cannot simply return the component. Try to just do this by mapping over the array. This will provide a cleaner and easier to implement solution. Don't spend too much time on this. I'd say give it about 10 mins and reach out if it's stumping you. Happy to walk you through it.

Reply:
I have a list with all team members' data and here need to get only full-stack developers' data, how can I do it without filtering?
Do you mean I need to create three lists (mentors, front-end, full stack) instead of one list for all team members?

Reply:
So you can still filter by using conditional logic instead of the filter method.